### PR TITLE
removing the args that i found they are not used

### PR DESCRIPTION
**Pull request by naahio**
removing the args that i found they are not used

### CODE REVIEW

File: tests/pyccel/scripts/test2.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        "shared_suffix": ".cpython-310-x86_64-linux-gnu.so",
+        "libs": [
+            "python3.10"
+        ],
+        "libdirs": [
+            "/usr/lib/python3.10/config-3.10-x86_64-linux-gnu",
+            "/usr/lib/x86_64-linux-gnu"
+        ]
+    }
+}
```
Comment (EmilyBourne) [inline tests/pyccel/scripts/test2.json:68]: Beware, these files are auto-generated but they should not be committed to the repo. Maybe you have some kind of setting which automatically commits any new file in the folder?

**Review by EniddeallA — APPROVED** [review]
Looks good!

File: pyccel/parser/base.py [hunk 2]
```diff
@@ -161,12 +160,11 @@ def __init__(self,
...
-        if headers:
-            if not isinstance(headers, dict):
-                raise TypeError('Expecting a dict of headers')
-
+        # if headers:
+        #     if not isinstance(headers, dict):
+        #         raise TypeError('Expecting a dict of headers')
 
-            self.scope.headers.update(headers)
+        #     self.scope.headers.update(headers)
```
Comment (EmilyBourne) [inline pyccel/parser/base.py:167]: Dead code should be removed

**Review by EmilyBourne — CHANGES_REQUESTED** [review]
You still don't seem to have completed your checklist for this PR

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ All notable changes to this project will be documented in this file.
 ### Added
 
 ### Fixed
-
+-   #1355 : remove BasicParser arguments.
```
Comment (EmilyBourne) [inline CHANGELOG.md:9]: I think this should go in "Changed" not "Fixed" as nothing was broken

Comment (naahio) [reply CHANGELOG.md:9]: ok

File: CHANGELOG.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ All notable changes to this project will be documented in this file.
 ### Fixed
 
 ### Changed
-
+-   #1355 : remove BasicParser arguments.
```
Comment (EmilyBourne) [inline CHANGELOG.md:11]: BasicParser is code so it should be in code quotes:
```suggestion
-   #1355 : Remove unused `BasicParser` arguments.
```

Comment (EmilyBourne) [reply CHANGELOG.md]: The start of sentences should be capitalised. I think it is also useful to mention that these arguments are unused so this is not a big change

File: CHANGELOG.md [hunk 5]
```diff
@@ -8,7 +8,7 @@ All notable changes to this project will be documented in this file.
 ### Fixed
 
 ### Changed
-
+-   #1355 : remove `BasicParser` arguments.
```
Comment (EmilyBourne) [inline CHANGELOG.md:11]: See previous comment. This should be capitalised really. I also think it is important to mention that these are unused arguments

Comment (naahio) [reply CHANGELOG.md:11]: ok

**Review by EmilyBourne — APPROVED** [review]

**Review by yguclu — APPROVED** [review]

**Pull request closed without merging**